ruleset.xml

```diff
@@ -10,4 +10,6 @@
     <rule ref="category/java/bestpractices.xml/UnusedLocalVariable" />
     <rule ref="category/java/bestpractices.xml/UnusedPrivateField" />
     <rule ref="category/java/bestpractices.xml/UnusedPrivateMethod" />
+
+    <rule ref="category/java/codestyle.xml/PackageCase" />
```

Comment:
Is there a specific reason to separate this rule from the others?

Reply:
The rule is part of the "codestyle" pmd ruleset (and the other rules are part of the "bestpractices" set). I just group by the rules by their origin.